src/main/java/net/spring/config/WebConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package net.spring.config;
+
+import net.spring.interceptor.StudentInterceptor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebConfiguration implements WebMvcConfigurer {
+    @Autowired
```

Comment:
It is better to use a constructor for dependency injection.
https://www.vojtechruzicka.com/field-dependency-injection-considered-harmful/

Reply:
As you use Lombok, please use Lombok annotation for a constructor.
https://www.baeldung.com/spring-injection-lombok
Example: @RequiredArgsConstructor(onConstructor = @__(@Autowired))